### PR TITLE
Clarify verbosity

### LINKED ISSUE #1080: Wrong documentation about verbose option
The [documentation](https://procrastinate.readthedocs.io/en/stable/howto/basics/command_line.html) says that `PROCRASTINATE_VERBOSE` env variable can be set to change the verbosity of the worker. It is even set in [some](https://github.com/procrastinate-org/procrastinate/blob/6c3e58248063c3c576ac37f054ba3055a3e819f8/tests/acceptance/conftest.py#L18) [tests](https://github.com/procrastinate-org/procrastinate/blob/6c3e58248063c3c576ac37f054ba3055a3e819f8/tests/integration/test_cli.py#L36). On the other hand the CLI just uses [`VERBOSE`](https://github.com/procrastinate-org/procrastinate/blob/6c3e58248063c3c576ac37f054ba3055a3e819f8/procrastinate/cli.py#L185-L186) env var.  Or is the `PROCRASTINATE_` prefix somehow added automagically?

The documentation also says that the verbosity is controlled by using `v` multiple times and that there are 3 levels. But there are actually four levels and those are controlled by an integer.

### PR DESCRIPTION
**Pull request by ewjoachim**
Closes #1080 

Ping @medihack (I hope I got your problem right)

### Successful PR Checklist:

- [ ] Tests
  - [x] (not applicable?)
- [x] Documentation
  - [ ] (not applicable?)

#### PR label(s): 
  - [ ] https://github.com/procrastinate-org/procrastinate/labels/PR%20type%3A%20breaking%20%F0%9F%92%A5
  - [ ] https://github.com/procrastinate-org/procrastinate/labels/PR%20type%3A%20feature%20%E2%AD%90%EF%B8%8F
  - [ ] https://github.com/procrastinate-org/procrastinate/labels/PR%20type%3A%20bugfix%20%F0%9F%95%B5%EF%B8%8F
  - [ ] https://github.com/procrastinate-org/procrastinate/labels/PR%20type%3A%20miscellaneous%20%F0%9F%91%BE
  - [ ] https://github.com/procrastinate-org/procrastinate/labels/PR%20type%3A%20dependencies%20%F0%9F%A4%96
  - [x] https://github.com/procrastinate-org/procrastinate/labels/PR%20type%3A%20documentation%20%F0%9F%93%9A

### CODE REVIEW

File: docs/howto/django/scripts.md [hunk 1]
```diff
@@ -8,11 +8,17 @@ The `DjangoConnector` configured on the default Procrastinate app cannot be
 used to run the worker directly but the `DjangoConnector` provides a method to get a
 suitable connector: `get_worker_connector()`.
 
+The script below calls `django.setup()` manually. Alternatively, you can use
+a [custom management command](https://docs.djangoproject.com/en/5.0/howto/custom-management-commands/)
+in which case you don't need to call `django.setup()`.
+
```
Comment (ewjoachim) [inline docs/howto/django/scripts.md:14]: (unrelated but I saw that wile browsing the django docs)

**Comment by medihack** [comment]
Yes, the documentation makes it pretty clear now. One thing that is still a bit confusing is the output of `./manage.py help procrastinate`:
```
usage: manage.py procrastinate [-h] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH] [--traceback] [--no-color] [--force-color] [--skip-checks] {worker,defer,healthchecks,shell} ...

Access procrastinate commands

positional arguments:
  {worker,defer,healthchecks,shell}
    worker              Launch a worker, listening on the given queues (or all queues). Values default to App.worker_defaults and then App.run_worker() defaults values.
    defer               Create a job from the given task, to be executed by a worker. TASK should be the name or dotted path to a task. JSON_ARGS should be a json object (a.k.a dictionary) with the job parameters
    healthchecks        Check the state of procrastinate
    shell               Administration shell for procrastinate

options:
  -h, --help            show this help message and exit
  --version             Show program's version number and exit.
  -v {0,1,2,3}, --verbosity {0,1,2,3}
                        Verbosity level; 0=minimal output, 1=normal output, 2=verbose output, 3=very verbose output
  --settings SETTINGS   The Python path to a settings module, e.g. "myproject.settings.main". If this isn't provided, the DJANGO_SETTINGS_MODULE environment variable will be used.
  --pythonpath PYTHONPATH
                        A directory to add to the Python path, e.g. "/home/djangoprojects/myproject".
  --traceback           Raise on CommandError exceptions.
  --no-color            Don't colorize the command output.
  --force-color         Force colorization of the command output.
  --skip-checks         Skip system checks.
```

For example, calling stuff like `./manage.py procrastinate --version` prints out the Django version. Of course, this is only minor stuff.

**Comment by ewjoachim** [comment]
You're in luck, we can actually [suppress](https://docs.djangoproject.com/en/5.0/howto/custom-management-commands/#django.core.management.BaseCommand.suppressed_base_arguments) those from the help message (they still exist on the command though). Just did.

**Comment by medihack** [comment]
> You're in luck, we can actually [suppress](https://docs.djangoproject.com/en/5.0/howto/custom-management-commands/#django.core.management.BaseCommand.suppressed_base_arguments) those from the help message (they still exist on the command though). Just did.

That's nice 🙂. I guess the rest of the options still make sense.

**Pull request merged**